data/users.yml

```diff
@@ -743,3 +743,8 @@ users:
     info: zuzuche.com 租租车
     website: https://www.zuzuche.com/
     logo: /images/users/zuzuche_logo.png
+
+  - name: bthhotels
+    info: bthhotels.com 首旅如家
+    website: https://www.bthhotels.com/
+    logo: /images/users/bthhotels_logo.png 
```

Comment:
I can't find this image in the PR.

Reply:
From the preview link(Deploy Preview), the logo can't show up.

<img width="896" alt="image" src="https://github.com/apache/skywalking-website/assets/5441976/a8f8f875-63f6-4e0d-b945-46910c005399">